API/NoDocs/Aspire.Azure.AI.OpenAI/AzureOpenAISettings.cs

```diff
@@ -0,0 +1,17 @@
+// Assembly 'Aspire.Azure.AI.OpenAI'
+
+using System;
+using System.Runtime.CompilerServices;
+using Aspire.Azure.Common;
+using Azure.Core;
+
+namespace Aspire.Azure.AI.OpenAI;
+
+public sealed class AzureOpenAISettings : IConnectionStringSettings
```

Comment:
Nit - Worth calling out that this pattern varies from the default of `<Namespace>Settings`.  I think this reads better than `AzureAIOpenAISettings` though and we should keep it.